neurom/fst/_mm.py

```diff
@@ -101,11 +101,6 @@ def map_sections(fun, nrn, neurite_type=NeuriteType.all):
                 for s in i_chain2(nrn.neurites, tree_filter=is_type(neurite_type)))
 
 
-def section_lengths(nrn, neurite_type=NeuriteType.all):
-    '''section lengths'''
-    return map_sections(mm.path_distance, nrn, neurite_type)
```

Comment:
Are you saying the tortuosity calculation is wrong? I went by the definition you gave in #375. Regarding the path distance to the root, there is another function for that. The definition of `mm.path_length` hasn't changed since the beginning of `NeuroM`.